MovieRecommender/pom.xml

```diff
@@ -66,8 +66,8 @@
       <scope>provided</scope>
       <exclusions>
         <exclusion>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>hadoop-client</artifactId>
+          <groupId>com.sun.jersey</groupId>
```

Comment:
jersey brings in asm which collide with ours and during unit tests when this dependencies is brought in it makes our services fails as we get handler through reflection.